docs/modules/ROOT/pages/udc.adoc

```diff
@@ -0,0 +1,228 @@
+= Universal Deployer Contract
+
+Account contracts make up very critical components of the StarkNet ecosystem, since a bug in any implementation—let alone a widespread one—could prove disastrous.
+Therefore, maximum caution is in order.
+In this spirit, we should consider trimming account responsibilities to simplify implementations and minimize bug/attack surfaces.
+
+This standard moves contract deployment responsibilities to an external, specialized deployer contract.
+The separation allows accounts to deploy contracts without compromising security.
+To remove the redundancy of deploying a new deployer contract for each account, the deployer contract should be a singleton Universal Deployer Contract (UDC).
+
+TIP: For further information on design decisions, see the https://community.starknet.io/t/universal-deployer-contract-proposal/1864[Universal Deployer Contract Proposal].
+
+== Table of Contents
+* <<quickstart, Quickstart>>
+* <<deploying_the_udc,Deploying the UDC>>
+* <<usage,Usage>>
+ ** <<reserved_address_space_for_deployer,Reserved Address Space for Deployer>>
+ ** <<reproducible_deployments,Reproducible Deployments>>
+* <<calculating_counterfactual_addresses,Calculating Counterfactual Addresses>>
+* <<universal_deployer_contract_api,Universal Deployer Contract API>>
```

Comment:
I would use `UDC API Specification` for consistency with other parts of the doc. A refactor to unify this pattern (and others) would be helpful IMO (out of scope for this PR).